src/biblenotify/core/notifications.py

```diff
@@ -96,7 +98,43 @@ def loadChapter(self, location: str) -> dict:
 
         contents_json = json.loads(contents_string)
 
+        text = self.highlightVerses(contents_json)
+
         return {
-            "text": contents_json["read"][0]["text"],
+            "text": text,
             "place": contents_json["read"][0]["chapter"]
         }
+
+    @Slot(str, result="QVariant")
+    def highlightVerses(self, contents_json):
+        # assuming every second verse is preceded by a special delimeter ;
+        # another delimeter should be used since this delimeter have too many edge cases
+        list = re.split(r";", verse["verse"])
+        start = list[0]
+        end =  list[-1]
+
+        # to find index of displayed verse from global dictionary {verse} in {contents_json}
+        slice1 = re.search(start, contents_json["read"][0]["text"]).span()
+        slice2 = re.search(end, contents_json["read"][0]["text"]).span()
+
+        to_bold = contents_json["read"][0]["text"][slice1[0] : slice2[1]]
+
+        # in case of two verses
```

Comment:
There are no longer any instances where the `verse to highlight is more than 1 verse, so you can remove any code related to 2-3 verses.